src/aind_metadata_service/labtracks/client.py

```diff
@@ -125,12 +125,15 @@ def get_subject_info(self, subject_id: str) -> ModelResponse:
         Parameters
         ----------
         subject_id: str
+            This is the id in the LabTracks ANIMALS_COMMON table
         """
+        if not subject_id.isalnum():
+            return ModelResponse.no_data_found_error_response()
```

Comment:
Does it suffice to use `ModelResponse.no_data_found_error_response()` or do we want to use a new `invalid_input_error_response`?

Reply:
no data found is fine